tests/e2e/specs/bulk-users-delete.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
+
+test.describe( 'Delete User', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		function generateRandomUser( index ) {
+			return {
+				username: `testuser${ index }_${ Math.floor(
+					Math.random() * 10000
+				) }`,
+				email: `test${ index }_${ Math.floor(
+					Math.random() * 10000
+				) }@gmail.com`,
```

Comment:
Why use an actual existing domain?
I'd use `domain.tld` to prevent that this test could ever possibly spam an actual email account.